app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -161,7 +161,7 @@ class AamvaStateMaintenanceWindow
       ],
       'UT' => [
         # 3rd Sunday of every month 1:00 am. to 9:00 am. ET
-        { cron: '0 1 0 0 Sun#3', duration_minutes: 8 * 60 },
+        { cron: '0 1 * * Sun#3', duration_minutes: 8 * 60 },
```

Comment:
🤦‍♂️ This meant it would run on the 0th day of the 0th month. Our calendar system predates Kernighan and Ritchie, so this was a nonsensical date.

The test failure shown was:

```
  2) Idv::AamvaStateMaintenanceWindow MAINTENANCE_WINDOWS consists of a cron expression and numeric minutes
     Failure/Error: expect(cron).to be_a(Fugit::Cron)
       expected nil to be a kind of Fugit::Cron
     # ./spec/services/idv/aamva_state_maintenance_window_spec.rb:84:in `block (5 levels) in <top (required)>'
     # ./spec/services/idv/aamva_state_maintenance_window_spec.rb:77:in `each'
     # ./spec/services/idv/aamva_state_maintenance_window_spec.rb:77:in `block (4 levels) in <top (required)>'
```